src/utils/constants.ts

```diff
@@ -289,12 +289,14 @@ export const MAP_TYPE_CONCEPT_SET = option("CONCEPT-SET");
 const CIEL_SOURCE_URL = "/orgs/CIEL/sources/CIEL/";
 const PIH_SOURCE_URL = "/orgs/PIH/sources/PIH/";
 const MSFOCP_SOURCE_URL = "/orgs/MSFOCP/sources/MSFOCP/";
+const MySource_SOURCE_URL = "/orgs/MySource/sources/MySource/";
```

Comment:
cc @hadijahkyampeire

Reply:
@HerbertYiga am missing your comment? I did that basing on this lick https://docs.google.com/presentation/d/1Q2yMyxTQrflQGc-O77KlGEXiPC_savon9ju9LDUaI5c/edit#slide=id.gd2498758a4_0_14 shared by Suruchi and the output is attached on the ticket.